app/views/user_mailer/reset_password_instructions.html.erb

```diff
@@ -2,7 +2,7 @@
   <table>
     <tbody>
     <tr>
-      <td width="140">
+      <td style="width:140px;">>
```

Comment:
Extra `>` snuck in here.

```suggestion
      <td style="width:140px;">
```

Reply:
resolved with commit 4d47dcd3eb355d26192a5753b38acca62d236617